client/lib/abtest/active-tests.js

```diff
@@ -45,6 +45,14 @@ module.exports = {
 		},
 		defaultVariation: 'original'
 	},
+	sidebarPlanLink: {
+		datestamp: '20160101',
+		variations: {
+			plans: 50,
+			'plans/my-plan': 50,
+		},
+		defaultVariation: 'plans'
```

Comment:
I disagree. Our concern is not better metrics, but a better product and I feel we should roll this for existing users also.
This significantly improves experience for premium users and our current users should also benefit from this.
I lowered treshold to 25%.